docs/electric-api.yaml

```diff
@@ -159,6 +159,23 @@ paths:
 
                 Must be provided as the `shape_id` parameter when making
                 subsequent requests where `offset` is not `-1`.
+            x-electric-schema:
```

Comment:
Have you considered how this would be structured when we add an include tree? Don't have to right now but I wonder if there's some top level `[{ table: public.foo, schema: {...}}]` wrapper structure that we could predict and thus bake in up front?

Reply:
Indeed, the `type` could be the name of the reference table and we need to ensure that we define the schema of the shape's root table and the schema of all referenced tables.